pydantic/error_wrappers.py

```diff
@@ -105,13 +105,21 @@ def flatten_errors(
 
 @lru_cache()
 def get_exc_type(cls: Type[Exception]) -> str:
-
-    base_name = 'type_error' if issubclass(cls, TypeError) else 'value_error'
```

Comment:
Here `else 'value_error'` clause causes all errors that are not `TypeError` or `ValueError` to be named as `value_error.<code>`. Now with addition of `AssertionError` I changed the logic a bit and it returns `<type/value/assertion>_error.<code>` if the exception class is a subtype of these errors. Is this the expected behavior?